README.md

```diff
@@ -40,10 +40,12 @@ class User(db.Model):
     username: Mapped[str] = mapped_column(unique=True)
 
 with app.app_context():
+    db.drop_all()
```

Comment:
@davidism I see that https://pypi.org/project/Flask-SQLAlchemy-Lite/ has the same example, so what do you think of the changes, as you'd probably want them there too? I do use drop_all in my non-Flask SQLAlchemy examples, as it does make them easier to re-run, there's just a slight risk of someone running that in production. But hopefully a very small risk as they need to change all this code.